tests/other_test.go

```diff
@@ -16,9 +17,36 @@ func TestRunBasicExample(t *testing.T) {
 		Prefix:             "mongodb-basic",
 		BestRegionYAMLPath: regionSelectionPath,
 		ResourceGroup:      resourceGroup,
+		CloudInfoService:   sharedInfoSvc,
 	})
 
 	output, err := options.RunTestConsistency()
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
+
+func testPlanICDVersions(t *testing.T, version string) {
```

Comment:
This was the naming convention used in other modules when we ran tests with multiple versions
https://github.com/terraform-ibm-modules/terraform-ibm-ocp-all-inclusive/blob/9864421b152c2a9dae6366b6ef704f265796cff3/tests/pr_test.go#L70


testPlanICDVersions is not the setup this is where the actual test runs.

Reply:
It looks like that example is less than ideal. `TestRunCompleteExample` uses `basicExampleTerraformDir`, so it is not complete, it is basic; which actually points to `examples/end-to-end-example`. So it is neither complete or basic.

Here `TestPlanICDVersions` uses `RunTestPlan` which is not used anywhere else I can find. Is the intention to only use a plan rather than apply to make sure it works? (The other calls are `RunTestConsistency` or `RunTestUpgrade`). It may be sufficient to just run a plan, but it would be nice to have comment in the code noting this, to avoid questions in the future (the referenced issue is good, but difficult to find longer term). A name like TestPlanBasic (avoiding the TestPlanBasicExample in other_tests.go) would be more helpful identifying what is actually running, but if a plan is sufficient and this is going to be consistent across other ICD modules this may be a good name.